tests/FakeItEasy.Tests.Approval.netstd/ApiApproval.cs

```diff
@@ -0,0 +1,33 @@
+namespace FakeItEasy.Tests.Approval
+{
+    using System;
+    using System.IO;
+    using System.Reflection;
+    using System.Runtime.CompilerServices;
+    using ApprovalTests;
+    using ApprovalTests.Reporters;
+    using PublicApiGenerator;
+    using Xunit;
+
+    public class ApiApproval
+    {
+        [Fact]
+        [UseReporter(typeof(DiffReporter))]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public void ApproveApiNetStd()
+        {
+            // Approvals and PublicApiGenerator aren't available for .NET Core, so
+            // we'll load the .NET Standard FakeItEasy assembly from disk and
+            // examine it.
+            string codeBase = Assembly.GetExecutingAssembly().CodeBase;
+            UriBuilder uri = new UriBuilder(codeBase);
+            string assemblyPath = Uri.UnescapeDataString(uri.Path);
+            var containingDirectory = Path.GetDirectoryName(assemblyPath);
+            var configurationName = new DirectoryInfo(containingDirectory).Name;
+            var assemblyFile = @"..\..\..\..\src\FakeItEasy.netstd\bin\" + configurationName + @"\FakeItEasy.dll";

```

Comment:
@thomaslevesque, I am not sure how to locate the assembly in a more straighforward way. On the brighter side, this won't break until we restructure the solution, at which time we'll likely be merging the two assemblies, and the whole point will become moot.

An alternative may be to just perform this test from the command line/build script and construct a utility that allows us to pass in the path of the assembly to approve, but we'd end up having to hardcode those paths or read them out of a nuspec or something.